src/Libraries/LibUtil.sol

```diff
@@ -23,11 +27,54 @@ library LibUtil {
         return addr == address(0);
     }
 
+    /// @notice Reverts the transaction with a given reason
+    /// @param data The revert reason
     function revertWith(bytes memory data) internal pure {
         assembly {
             let dataSize := mload(data) // Load the size of the data
             let dataPtr := add(data, 0x20) // Advance data pointer to the next word
             revert(dataPtr, dataSize) // Revert with the given data
         }
     }
+
+    /// @notice Converts an address type to a bytes32 type
+    /// @param addr The address to be converted
+    /// @return address in bytes32 format
+    function convertAddressToBytes32(
```

Comment:
do we also want to be more explicit and split it to `convertAddressToBytes32WithLeftPaddedZeros` and `convertAddressToBytes32WithRightPaddedZeros` with `bytes32(bytes20(uint160(addr)))`

Reply:
I thought we discussed to have one standard way in our codebase (which is this one)
The other one is a special case that is only used by relay. Do you think it makes sense to have helper functions for this purpose?

Reply:
that's true. Ok let's keep it like this!